Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/dialogs/GameDetailsDialog.java

```diff
@@ -0,0 +1,74 @@
+package org.dolphinemu.dolphinemu.dialogs;
+
+import android.app.AlertDialog;
+import android.app.Dialog;
+import android.os.Bundle;
+import android.support.design.widget.FloatingActionButton;
+import android.support.v4.app.DialogFragment;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.squareup.picasso.Picasso;
+
+import org.dolphinemu.dolphinemu.R;
+import org.dolphinemu.dolphinemu.activities.EmulationActivity;
+import org.dolphinemu.dolphinemu.model.GameFile;
+import org.dolphinemu.dolphinemu.services.GameFileCacheService;
+import org.dolphinemu.dolphinemu.utils.PicassoUtils;
+
+public final class GameDetailsDialog extends DialogFragment
+{
+  private static final String ARG_GAME_PATH = "game_path";
+
+  public static GameDetailsDialog newInstance(String gamePath)
```

Comment:
It feels a bit weird that this has a factory method, but still leaves the default constructor `public`. The `GamePropertiesDialog` (former `GameSettingsDialog`) does the same, but this doesn't necessarily mean its good, right? Or is there a hidden requirement somewhere in the Android UI that I'm not aware of?

Reply:
I'm not sure. I didn't write this code, I just reverted the removal.

Reply:
Android requires fragments have a public constructor. When fragments are recreated by the OS (generally after being destroyed in a lowmemory situation) that constructor is used, and then the argument bundle supplied inside this method is used to help recreate the previous condition of the destroyed fragment.